src/scss/typography.scss

```diff
@@ -1,3 +1,8 @@
+// Base font sizing
+html {
+  font-size: 62.5%; // Adjusted so that 36px is equal to 3.6rem
```

Comment:
As mentioned in another comment. I don't think this is a good rule to put in. Also the comment assumes consistent browser font sizing.

Reply:
Agree, I think this isn't using the default bootstrap settings properly.